### PR TITLE
Peach

### PR DESCRIPTION
**Pull request by Rfpnj**
Adds a peach item, that increases ass size by one. Image needs tweaking to fit correctly, and for lore purposes. I did not add it to drop tables. Has been tested as working as intended.

### CODE REVIEW

**Comment by Innoxia** [comment]
Thank you for this! ^^

I might use this for the item modding template example, if that's ok with you? :3

**Comment by Innoxia** [comment]
Ok, will do! Thank you! ^^

**Comment by Rfpnj** [comment]
Had to go back and make some more edits to reflect changes to listvalues

File: src/com/lilithsthrone/game/inventory/item/ItemType.java [hunk 1]
```diff
@@ -4020,6 +4020,44 @@ public String getUseDescription(GameCharacter user, GameCharacter target) {
 		}
 	};
 	
+	public static AbstractItemType PEACH = new AbstractItemType(
+			25,
+			null,
+			false,
+			"Peach",
+			"Peaches",
+			"A delicate fruit often cultivated in temperate climates. Also it kind of looks like an booty if you squint.",
```
Comment (Pimvgd) [inline src/com/lilithsthrone/game/inventory/item/ItemType.java]: an booty?

File: src/com/lilithsthrone/game/inventory/item/ItemType.java [hunk 2]
```diff
@@ -4020,6 +4020,44 @@ public String getUseDescription(GameCharacter user, GameCharacter target) {
...
+		public String getUseName() {
+			return "eat";
+		}
+		
+		@Override
+		public String getUseDescription(GameCharacter user, GameCharacter target) {
+			return getGenericUseDescription(user, target,
+					"You eat the peach. The sweet taste of rapture overwhelmes you.",
+					"You force [npc.name] to eat the peach. The sweet taste of rapture overwhelmes them.",
+					"[npc.Name] produces an peach, and then proceeds to eat it. The sweet taste of rapture overwhelmes you both.",
```
Comment (Pimvgd) [inline src/com/lilithsthrone/game/inventory/item/ItemType.java]: an peach?

**Comment by lllusion3469** [comment]
Something seems to have messed up the peach.svg file, because instead of an actual svg file it's an HTML file containing the Github file view of peach.svg.

**Comment by Innoxia** [comment]
As Illusion3469 says, the peach.svg file is completely broken. :s

File: src/com/lilithsthrone/game/inventory/item/ItemType.java [hunk 3]
```diff
@@ -4020,6 +4020,44 @@ public String getUseDescription(GameCharacter user, GameCharacter target) {
...
+			return "eat";
+		}
+		
+		@Override
+		public String getUseDescription(GameCharacter user, GameCharacter target) {
+			return getGenericUseDescription(user, target,
+					"You eat the peach. The sweet taste of rapture overwhelmes you.",
+					"You force [npc.name] to eat the peach. The sweet taste of rapture overwhelmes them.",
+					"[npc.Name] produces a peach, and then proceeds to eat it. The sweet taste of rapture overwhelmes you both.",
+					"[npc.Name] produces a peach, and then proceeds to force you to eat it. The sweet taste of rapture overwhelmes you both.");
```
Comment (picobyte) [inline src/com/lilithsthrone/game/inventory/item/ItemType.java]: suggestion: replace 'proceeds to force you' by 'forces you'

**Comment by Innoxia** [comment]
I've adapted this and added it into the game on my end (and credited you in the patch notes), so I'll close this now. Thank you for the PR and sorry it took me so long to add it to the game!

**Pull request closed without merging**